OsmAnd/src/net/osmand/plus/resources/RegionAddressRepositoryBinary.java

```diff
@@ -229,35 +271,11 @@ public synchronized void preloadStreets(City o, ResultMatcher<Street> resultMatc
 				}
 			}
 
-			int initialsize = citiesToFill.size();
```

Comment:
I fixed this: https://github.com/osmandapp/Osmand/commit/ed5b746392548861f9b5bbdb4842de6d5dfaa1af